build-system/tasks/process-github-issues.js

```diff
@@ -51,12 +50,23 @@ const milestoneOptions = {
 
 // 4 is the number for Milestone 'Backlog Bugs'
 const MILESTONE_BACKLOG_BUGS = 4;
+// 11 is the number for Milestone '3P Implementation'
+const MILESTONE_3P_IMPLEMENTATION = 11;
+// 12 is the number for Milestone 'Docs Updates'
+const MILESTONE_DOCS_UPDATES = 12;
 // By default we will assign 'Pending Triage' milestone, number 20
 const MILESTONE_PENDING_TRIAGE = 20;
+// 22 is the number for Milestone 'Prioritized FRs'
+const MILESTONE_PRIORITIZED_FRS = 22;
 // 23 is the number for Milestone 'New FRs'
 const MILESTONE_NEW_FRS = 23;
-// 12 is the number for Milestone 'Docs Updates'
-const MILESTONE_DOCS_UPDATES = 12;
+// 25 is the number for Milestone 'Great First Issues (GFI)'
+const MILESTONE_GREAT_ISSUES = 25;
+// days for biweekly updates
+const BIWEEKLY_DAYS = 14;
+// days for quarterly updates
+const QUARTERLY_DAYS = 89;
+const NUM_BATCHES = 11;
 
```

Comment:
Add a comment above this, mentioning the current default batch size and the total number of issues that will be fetched.

Reply:
done :)